pyproject.toml

```diff
@@ -1,15 +1,188 @@
 [build-system]
 requires = [
```

Comment:
I think we should try to reduce this. Most of them are needed because of `bundle_resources`, and in the future, this can be improved. Some of the packages I feel can be removed:  

- `hatch-conda` is this needed during the build step?
- `bleach` can see this import can be moved into methods and removed from here. 
- `requests` maybe use `urllib.request` instead of. 
- `tqdm` can be removed if we make a file like `panel/widgets/_tqdm_indicator.py` and guard the import in `panel/widgets/indicators.py`.